bears/swift/TailorBear.py

```diff
@@ -0,0 +1,80 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.results.Result import Result
+from coalib.settings.Setting import path
+
+
+@linter(executable="tailor",
+        output_format=None,
+        prerequisite_check_command=("tailor", "-v"),
+        prerequisite_check_fail_message="Tailor is not installed. Refer "
+        "https://tailor.sh/ for installation details.")
+class TailorBear:
+    """
+    Analyze Swift code using ``tailor`` and check for code style related
+    warning messages.
+
+    For more information on the analysis visit <https://tailor.sh/>
+    """
+    LANGUAGES = "Swift"
+
+    severity_map = {"warning": RESULT_SEVERITY.NORMAL,
+                    "error": RESULT_SEVERITY.MAJOR}
+
+    def process_output(self, output, filename, file):
+        output = json.loads(output)
+
+        for item in output["files"][0]["violations"]:
+            column = item["location"]["column"] if "column" in item[
+                "location"].keys() else None
+
+            yield Result.from_values(
+                origin="{} ({})".format(self.name, item["rule"]),
+                message=item["message"],
+                file=filename,
+                line=item["location"]["line"],
+                column=column,
+                severity=self.severity_map[item["severity"]])
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         max_line_length: str="80",
```

Comment:
why are those str and not int? Please take an int and convert them to str later as needed. This way the type gets validated.